File: Cards.SPA/src/app/_models/response-models/dictionary-ok-response.ts
import { OkResponse } from "./ok-response";
import { Category } from '../category';
import { Mode } from '../mode';
import { RepeatRate } from '../repeat-rate';
import { Card } from '../card';

export class DictionaryOkResponse extends OkResponse<Card>{
    categories: Category[] = [];
    modes: Mode[] = [];
    repeatRates: RepeatRate[] = [];
}

File: Cards.Core/Models/OkResponses/DictionaryOkResponse.cs
using System.Collections.Generic;


namespace Cards.Core.Models.OkResponses
{
    public class DictionaryOkResponse : OkResponse<CardModel>
    {
        public IEnumerable<CategoryModel> Categories { get; set; }
        public IEnumerable<ModeModel> Modes { get; set; }
        public IEnumerable<RepeatRateModel> RepeatRates { get; set; }
    }
}